src/main/java/org/variantsync/diffdetective/diff/difftree/serialize/nodeformat/TypeDiffNodeFormat.java

```diff
@@ -3,12 +3,12 @@
 import org.variantsync.diffdetective.diff.difftree.DiffNode;
 
 /**
- * Labels are of the form <code>CodeType_DiffType</code>.
+ * Labels are of the form <code>NodeType</code>.
```

Comment:
This comment is also lying now. While renaming `CodeType` to `NodeType` I seemingly got this comment wrong.

Reply:
Oh, lol, these comments where also lying before this edit because they got the order wrong. :laughing: